multi-node-k8s.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+set -e -o pipefail
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//')"
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible \
+	golang
```

Comment:
why we need golang?

Reply:
the tests get executed from the host, and that means compiling the e2e.test executable

other dependencies for the compiling of ceph-csi need to be installed too, see [the docs](/ceph/ceph-csi/blob/master/docs/development-guide.md#workspace-and-repository-setup)

Maybe it is easier to add a `make containerized-build TARGET=e2e.test` option that compiles the executable in a container. It can then be run on the host without the need to install the build tools

Reply:
no need to install golang, nor libcephfs-devel librbd-devel librados-devel further below.